AnastasiaB/cards4-template.js

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
Work here looks great. Clear to read and works. Find one comment below on your fullset array, which needs just a minor adjustment.